src/renderer/components/scroll-spy/__tests__/scroll-spy.test.tsx

```diff
@@ -124,17 +117,14 @@ describe("<TreeView/> dataTree inside <ScrollSpy/>", () => {
       />
     ));
 
-    await waitFor(() => {
-      expect(queryByTitle("Application")).toBeInTheDocument();
-      expect(queryByTitle("Appearance")).toBeInTheDocument();
-      expect(queryByTitle("Theme")).toBeInTheDocument();
-    });
+    expect(await screen.findByTitle("Application")).toBeInTheDocument();
+    expect(await screen.findByTitle("Appearance")).toBeInTheDocument();
+    expect(await screen.findByTitle("Theme")).toBeInTheDocument();
```

Comment:
Nit: Multiple expects in test prevent name of test documenting the exact reason for test to fail.

Reply:
Resolved for being just a nit. Worth noting tho'.